### ISSUE TITLE
mixer needs a way to negate trim offset from input

### ISSUE THREAD

**Issue by arronbates** [opentx/opentx#869]
From a discussion on RCGroups...

current input screens pass the offset from trims to the mixers, which is fine, although the mixer needs a way to negate the offset from trim.

For example, people should put their dual rates and expo through the input screen because that is what it is intended to do. We also need to have trims active because, we naturally need trims. However, if you need to set up a coupling mix from rudder -> elevator, in the mixer you should put "IRud" (to make sure it gets the effects of expo and rates, naturally), but if you do this, any trims on rudder will mean you will have to re-trim your elevator.

In short, mixers need a way to get the output from the input screen (curves, etc), but without the trim. One idea would be to allow the offset in mixer screen to get "-Trim" in an equivalent way that it can be "-GV" to negate the trim offset.

Maybe a more powerful route/feature could be to treat all trims as global variables?... allowing a trim to be assignable wherever a global variable is assignable, assign its inverse, etc.
